.github/workflows/live-links.yaml

```
@@ -39,7 +39,7 @@ jobs:
             http://www.adlinktech.com/en/index
             /flags.png
             /flags@2x.png
-            http://*.example.com
+            http:\/\/[a-z0-9]*\.example\.com
```

should it be case insensitive as well? 
```suggestion
            http:\/\/[a-z0-9]*\.example\.com/i
```

---

Could be I guess